test/measurement/discriminator/test_iq_discriminator.py

```diff
@@ -0,0 +1,89 @@
+import unittest
+import random
+
+import qiskit
+from qiskit import Aer
+from qiskit.ignis.measurement.discriminator.filters import DiscriminationFilter
+from qiskit.ignis.measurement.discriminator.iq_discrimination import \
+    LinearIQDiscriminationFitter
+from qiskit.ignis.mitigation.measurement import circuits
+from qiskit.result.models import ExperimentResultData
+
+
+class TestLinearIQDiscriminator(unittest.TestCase):
+
+    def setUp(self):
+        self.shots = 52
+        self.qubits = [0, 1]
+
+    def test_discrimination(self):
+        meas_cal, state_labels = circuits.tensored_meas_cal([[0], [1]])
+
+        backend = Aer.get_backend('qasm_simulator')
+        job = qiskit.execute(meas_cal, backend=backend, shots=self.shots,
+                             meas_level=1)
+
+        cal_results = job.result()
+
+        # Make up some fake data for the qubits
+        def qubit_shot(i_center, q_center, std):
+            return [i_center + random.gauss(0, std),
+                    q_center + random.gauss(0, std)]
+
+        def create_shots(i_center, q_center):
```

Comment:
See above comment about placing this method somewhere in a test utils module for reuse in examples.